example/hyprland.desktop

```diff
@@ -2,4 +2,7 @@
 Name=Hyprland
 Comment=An intelligent dynamic tiling Wayland compositor
 Exec=Hyprland
-Type=Application
\ No newline at end of file
+Type=Application
+X-LightDM-DesktopName=Hyprland
```

Comment:
imo we shouldn't add this as LightDM is known not to work with Wayland anyway.

Reply:
LGTM, overwrote the commit and removed `X-LightDM-DesktopName`